cargo-dist-schema/src/lib.rs

```diff
@@ -610,12 +621,21 @@ impl DistManifest {
         if let Some(position) = self.releases.iter().position(|r| r.app_name == name) {
             &mut self.releases[position]
         } else {
-            let install_dir_env_var =
-                Some(name.to_ascii_uppercase().replace('-', "_") + "_INSTALL_DIR");
+            let env_app_name = name.to_ascii_uppercase().replace('-', "_");
+            let install_dir_env_var = format!("{env_app_name}_INSTALL_DIR");
+            let unmanaged_dir_env_var = format!("{env_app_name}_UNMANAGED_INSTALL");
+            let disable_update_env_var = format!("{env_app_name}_DISABLE_UPDATE");
```

Comment:
not DISABLE_UPDATER?

Reply:
my thought here was that the user cares about the "update" behavior. there is not always an "updater" so "update" focused on the behavior the user understands, not the impl